src/Makefile.in

```diff
@@ -261,6 +261,10 @@ stratcon_datastore.o:	stratcon_datastore.c
 		echo "- compiling $<" ; \
 		$(CC) $(CPPFLAGS) $(CFLAGS) -c $< ; \
 	fi
+	
+noit_metric_rollup.o:  
+  $(CC) -fPIC $(CPPFLAGS) $(CFLAGS) -c noit_metric_rollup.c
+	
```

Comment:
I needed -fPIC but I accidentally built the .o files instead of the .lo one
Now it's not needed any more